adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -19,15 +19,20 @@ package org.bdgenomics.adam.rdd
 
 import java.io.FileNotFoundException
 import java.util.regex.Pattern
-import htsjdk.samtools.SAMFileHeader
+
```

Comment:
hm yea, intellij "optimized" these imports when I accidentally hit ⌘⇧O at some point, and it does blank lines between java stdlib, other, and scala stdlib imports, I guess. In any case I'll remove it.